tool/actions-plan-preview/DOCKER_BUILD

```diff
@@ -1,2 +1,2 @@
-version: 1.7.2
+version: 1.7.3
```

Comment:
@gkuga Thank you. Could you update this to `1.8.0`?

Reply:
I updated the version to 1.8.0 👍